common/src/types/tasks.rs

```diff
@@ -44,7 +44,6 @@ pub struct QueuedTask {
 /// TODO: We can add completed and failed states if/when we implement a task
 /// history
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
-#[serde(tag = "status")]
```

Comment:
`bincode` doesn't support tagged enums, the tag was for the FE, but we're planning to refactor FE task state information anyways.